.github/workflows/run_linters.yml

```diff
@@ -58,6 +58,9 @@ jobs:
       - name: Give Linters A Go
         id: linter-setup
         run: ":"
+      - name: Ensure genesis_call.dme is unchanged
+        if: steps.linter-setup.conclusion == 'success' && !cancelled()
+        run: sha256sum -c tools/ci/genesis_call.dme.sha256sum
```

Comment:
Split this into a .sh file task in the CI folder so we can emit a verbose error message on how to fix this failure (removing changes to the file if accidental, regenerating the sha256 if intentional).